ansible/group_vars/all.example

```diff
@@ -18,5 +18,9 @@ nginx_try_files: /index.php$is_args$args
 # Node Version
 node_version: stable
 
+# Timeout
+fastcgi_read_timeout: 300
```

Comment:
For the base config, can we set the default values to those specified by `nginx` (60 not 300)